src/config/configAggregator.ts

```diff
@@ -158,23 +174,22 @@ export class ConfigAggregator extends AsyncOptionalCreatable<ConfigAggregator.Op
    *
    * @param key The config key.
    */
-  public static getValue(key: string): ConfigInfo {
-    return this.getInstance().getInfo(key);
+  public static getValue(key: string, projectPath?: string): ConfigInfo {
+    return this.getInstance(projectPath ?? process.cwd()).getInfo(key);
   }
 
   /**
    * Get the static ConfigAggregator instance. If one doesn't exist, one will be created with
    * the **encrypted** config values. Encrypted config values need to be resolved
    * asynchronously by calling {@link ConfigAggregator.reload}
    */
-  // Use typing from AsyncOptionalCreatable to support extending ConfigAggregator.
-  // We really don't want ConfigAggregator extended but typescript doesn't support a final.
-  private static getInstance<P, T extends AsyncOptionalCreatable<P>>(this: new () => T): T {
-    if (!ConfigAggregator.instance) {
-      ConfigAggregator.instance = new this();
-      (ConfigAggregator.instance as ConfigAggregator).loadPropertiesSync();
+  private static getInstance(projectPath = process.cwd()): ConfigAggregator {
+    if (!ConfigAggregator.instances.has(projectPath)) {
```

Comment:
I think this has the potential to be sketchy, even before this PR.

because getValue is sync, we can't do the real `create` here.  So I think it's going to have a ConfigAggregator missing the options.configMeta stuff?

Reply:
Agreed - but I think that's why the jsdoc points out that you should call `reload` after